doc: build: sysbuild: Add documentation on image setup (#86129)
zephyrproject-rtos/zephyr: doc/develop/application/index.rst

   When using ``-DBOARD_ROOT=<board-root>`` both absolute and relative paths can
   be used. Relative paths are treated relatively to the application directory.

.. note::

>    This is not supported when using sysbuild, instead the ``BOARD_ROOT`` can be defined in
>    a module or sysbuild CMake file, see :ref:`sysbuild_board_roots` for details.

Reviewer: How about:
```suggestion
   When using sysbuild, then the ``BOARD_ROOT``must defined in
   a module or in the sysbuild CMakeLists.txt file, see :ref:`sysbuild_board_roots` for details.
```

Author: Updated
